### PR TITLE
Updating the documentation

### PR DESCRIPTION
**Pull request by TharukaCkasthuri**

### CODE REVIEW

File: docs/docs/instalation/1_installing_jaseci.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Jaseci Installation Guide
```
Comment (TharukaCkasthuri) [inline docs/docs/instalation/1_installing_jaseci.md]: - contents from docs\docs\getting-started\installation_intro.md
- contents from docs\docs\getting-started\installation.md
- contents from support\guide\getting_started\installation.md

File: docs/docs/intro/1_what_and_why_jaseci.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# What and Why Jaseci
```
Comment (TharukaCkasthuri) [inline docs/docs/intro/1_what_and_why_jaseci.md:1]: - content from introduction/what_and why_jaseci.md

File: docs/docs/intro/2_abstractions_of_jaseci.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Abstractions of Jaseci
```
Comment (TharukaCkasthuri) [inline docs/docs/intro/2_abstractions_of_jaseci.md]: graphs

- content from introduction/abstractions_of_jaseci.md

walkers

- content from jaseci/docs/docs/jaseci_architype/walkers.md
- content from introduction/abstractions_of_jaseci.md
- content from jaseci\docs\docs\jaseci_architype\walkers_explanation.md
- content from jaseci\examples\CanoniCAI\codelabs\lang_docs\walkers.md


abilities

- content from examples\CanoniCAI\codelabs\lang_docs\abilities.md

here and visitor
- content from examples\CanoniCAI\codelabs\lang_docs\here_visitor.md

Actions
- content from examples\CanoniCAI\codelabs\lang_docs\actions.md

**Comment by ypkang** [comment]
I believe this is obsolete now given the new outline we merged. Closing. @TharukaCkasthuri

**Pull request closed without merging**